src/Microsoft.ML.Parquet/ParquetLoader.cs

```diff
@@ -33,7 +33,7 @@ namespace Microsoft.ML.Data
     /// Loads a parquet file into an IDataView. Supports basic mapping from Parquet input column data types to framework data types.
     /// </summary>
     [BestFriend]
-    internal sealed class ParquetLoader : IDataLoader, IDisposable
+    internal sealed class ParquetLoader : ILegacyDataLoader, IDisposable
```

Comment:
>ILegacyDataLoader [](start = 42, length = 17)

ah! I cant see ParquetLoader going to legacy, so this will need to be updated to DataLoader right? Are there any issues tracking this work? I would think this could happen post 1.0 but would still be good to track.

Reply:
Will open an issue regarding this. You are right, I wonder if Yael's work related to #2735 will touch this.

---
In reply to: [260398136](https://github.com/dotnet/machinelearning/pull/2731#discussion_r260398136) [](ancestors = 260398136)